miracl/reg/miracl_reg_clar-allen.sh

```diff
@@ -463,7 +463,7 @@ function resampleclar()
 	ifdsntexistrun ${resclar} "Resampling CLARITY input" \
 	ResampleImage 3 ${inclar} ${resclar} ${vox}x${vox}x${vox} ${ifspacing} ${interp}
 
-	c3d ${resclar} -type ushort -o ${resclar}
+	c3d ${resclar} -type ushort -o /code/clar.nii.gz
```

Comment:
What is this doing again? Is it saving sth into `/code`? Also, do we have an env var for `/code`? If not, should we create one to use here?

Reply:
this is just a temporary change. Registration is giving us problems so needed to check intermediate outputs. Should have done it on a branch not already in a PR...

Reply:
Ah of course, I could have guessed that as well that it's debugging...do you want to hop on Zoom re: the reg issue?